src/documentmanager/documentmanager.cpp

```diff
@@ -532,27 +545,42 @@ void DocumentManager::closeEmptyDocArea()
     }
 }
 
-void DocumentManager::closeCurrentDocArea()
+void DocumentManager::configureDocuments(std::function<void(DocumentManager*, CodeEditor*, const QString&)> functor,
+                                        const QString &newValue)
 {
-    if (mDocAreas.size() < 2)
+    // applying new settings to every opened doc view
+    for (const auto& area: mDocAreas)
     {
-        return;
-    }
-
-    // close area in focus
-    auto pAreaInFocus = areaInFocus();
+        auto windowsList = area->subWindowList();
 
-    if (!pAreaInFocus)
-    {
-        return;
+        std::for_each(windowsList.begin(), windowsList.end(),
+                      [this, &functor, &newValue](const auto& wdw)
```

Comment:
[this, &functor, &newValue](const auto **&**wdw)

Reply:
fixed